tests/integration/test_plugins.py

```diff
@@ -39,7 +39,7 @@ class MockConfig:
 
         plugins_dir = PLUGINS_TEST_DIR
         plugins_openai = [PLUGIN_TEST_OPENAI]
-        plugins_denylist = ["AutoGPTPVicuna"]
+        plugins_denylist = ["AutoGPTPVicuna", "auto_gpt_guanaco"]
```

Comment:
Is this the correct formatting for the name of the plugin?

Reply:
Yes, the value given here should be the module name. `AutoGPTPVicuna` is the class name so it's incorrect. It should be `lower_snake_case`.

I would change the existing test plugin, but it's dependent on some serious issues with the zip-based plugin code that I don't think I should touch.